riocli/apply/__init__.py

```diff
@@ -35,11 +35,11 @@
               help='Dry run the yaml files without applying any change')
 @click.option('--show-graph', '-g', is_flag=True, default=False,
               help='Opens a mermaid.live dependency graph')
-@click.option('--values', '-v',
+@click.option('--values', '-v', multiple=True, default=(),
               help="Path to values yaml file. Key/values "
```

Comment:
suggestion: Can we update help to also mention multiple values are possible? Also, in the docstring example, we can have one multiple value example.